CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 - Exports `VideoPlayerAspectRatios`
 - Adds Tugboat QA configuration for PR preview deployments to replace Netlify.
+- Updates the `DatePicker`, `TextInput`, and `Label` components to pass an optional `showOptReqLabel` prop to conditionally render "Required"/"Optional" in the label text.
```

Comment:
This same pattern with `Label` is also used on `Select`, so `Select` will need to be listed here and the `Select` component will also need to be updated.

Reply:
That is true. I'd argue that it should be its own ticket to keep the scope of this PR for the `DatePicker` component. Updating the `Select` component to follow this pattern will not be difficult to do.

Reply:
OK, makes sense.  I saw the new ticket for that task.